### PR TITLE
migrated web_dashboard_tile module to version 12

### PR DESCRIPTION
**Pull request by steigendit**
migrated web_dashboard_tile module to version 12

### CODE REVIEW

**Review by legalsylvain — CHANGES_REQUESTED** [review]
Hi @steigendit. Thanks for porting the module. 
- same comment as @tarteo said here. https://github.com/OCA/web/pull/1355#issuecomment-523040308
- tested on runbot. Doesn't seems to work.

I'd like to make some improvement for the 12.0 version. could you ping we when your PR is OK ? 

thanks !

File: web_dashboard_tile/__init__.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
```
Comment (daramousk) [inline web_dashboard_tile/__init__.py:1]: Remove encodings from everywhere.

File: web_dashboard_tile/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# © 2010-2013 OpenERP s.a. (<http://openerp.com>).
```
Comment (daramousk) [inline web_dashboard_tile/__manifest__.py:2]: Replace the copyright symbol with `Copyright`, remove encodings

File: web_dashboard_tile/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    'data': [
+        'security/ir.model.access.csv',
+        'security/rules.xml',
+        'data/res_groups.xml',
+        'data/tile_tile.xml',
+        'views/tile.xml',
+        'views/templates.xml',
+        
+    ],
+    'demo': [
```
Comment (daramousk) [inline web_dashboard_tile/__manifest__.py:34]: If there are no demo data, just remove the demo key.

File: web_dashboard_tile/data/res_groups.xml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
```
Comment (daramousk) [inline web_dashboard_tile/data/res_groups.xml:3]: There is no need for `data`, remove commented out yaml.

File: web_dashboard_tile/static/src/js/custom_js.js [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//    GNU Affero General Public License for more details.
+//
+//    You should have received a copy of the GNU Affero General Public License
+//    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+//#############################################################################
+odoo.define('web_dashboard_tile', function (require) {
+"use strict";
+//odoo.web_dashboard_tile = function (instance)
```
Comment (daramousk) [inline web_dashboard_tile/static/src/js/custom_js.js:23]: Remove all these, replace copyright header.

**Review by daramousk — CHANGES_REQUESTED** [review]
Thank you for this,

before I continue with my review I think you need to verify this actually works to a degree and not only that it's installation does not fail, let us know when you do so that I continue reviewing

**Comment by legalsylvain** [comment]
Closing due to no response to fix the problems.
Seems draft PR, regarding log in javascript.

Superseded by https://github.com/OCA/web/pull/1428

**Pull request closed without merging**